test/expected/hnsw_insert_array.out

```diff
@@ -147,9 +161,9 @@ SELECT id, ROUND((v <-> '{1,0,0,0,0,0,21,35,1,0,0,0,0,77,51,42,66,2,0,0,0,86,140
   39 | 56265.00
  886 | 71637.00
  402 | 77013.00
- 518 | 85024.00
  340 | 87261.00
  331 | 87796.00
+ 518 | 85024.00
```

Comment:
is this fine?

Reply:
I think we overlooked an issue in a previous PR (#18) where the regression tests for `real[]` column typed tables did not actually create an index.
It seems Varik fixed it there, which caused some of the changes.

Without the index we do exact ordering, which is always 100% accurate. With the index, we do approximate ordering so anomalies like the above are acceptible.